src/client/client.go

```diff
@@ -308,11 +308,6 @@ func getUserMachineAddrAndOpts(context *config.Context) (*grpcutil.PachdAddress,
 		if err != nil {
 			return nil, nil, fmt.Errorf("could not parse 'PACHD_ADDRESS': %v", err)
 		}
-
-		if envAddr.Secured {
-			options = append(options, WithSystemCAs)
-		}
```

Comment:
Is this something we should just be removing?  This seems like a good thing to have, and if it isn't hooked up at the moment, should we fix it?

Reply:
I asked @ysimonson who wrote this code about it and he said it should just be removed because the only case the function supports shouldn't be using SystemCAs. I think it maybe made sense in a previous version of the function.